BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -944,6 +944,15 @@
 "SetUpUnlock" = "Set up unlock";
 "SetUpBiometricsOrChooseAPinCodeToQuicklyAccessYourVaultAndAutofillYourLogins" = "Set up biometrics or choose a PIN code to quickly access your vault and autofill your logins.";
 "SetUpLater" = "Set up later";
+"ChooseYourMasterPassword" = "Choose your master password";
+"ChooseYourMasterPasswordMessage" = "Choose a unique and strong password to  keep your information safe.";
+"MasterPasswordRequiredTitle" = "Master password (required)";
+"RetypeMasterPasswordTitle" = "Re-type master password (required)";
+"MasterPasswordHintTitle" = "Master password hint";
+"BitwardenCannotResetLostPasswordMessage" = "Bitwarden cannot reset a lost or forgotten master password.";
+"LearnAboutWaysToPreventAccountLockout" = "Learn about ways to prevent account lockout";
+"WhatMakesAPasswordStrong" = "What makes a password strong?";
+"Characters" = "%1$@ Characters";
```

Comment:
⚠️ Change key name to `XCharacters` because if we were to need the string "Characters" without the number it would be already taken by this.
```suggestion
"XCharacters" = "%1$@ Characters";
```